//Application with out try catch
public class exception2 
{

	public static void main(String[] args)
	{
		System.out.println(1);    //After line 7 data is not getting printed only 1 will print
		System.out.println(10/0);
		System.out.println(3);
		System.out.println(4);
		System.out.println(5);

	}

}
